src/OpenDiffix.Core/Aggregator.fs

```diff
@@ -124,14 +124,8 @@ type private DiffixCount() =
 
       match aidValue with
       // No AIDs, add to unaccounted value
-      | Null
-      | Value.List [] -> aidState.UnaccountedFor <- aidState.UnaccountedFor + valueIncrease
-      // List of AIDs, distribute contribution evenly
-      | Value.List aidValues ->
-        let partialIncrease = (float valueIncrease) / (aidValues |> List.length |> float)
-
-        aidValues
-        |> List.iter (fun aidValue -> increaseContribution partialIncrease aidValue aidContributions)
+      | Null -> aidState.UnaccountedFor <- aidState.UnaccountedFor + valueIncrease
+      | Value.List _ -> failwith "Lists of AIDs and distributing of contributions are not supported."
```

Comment:
Why is this union case kept if it is not supported anymore?

Reply:
You mean to remove `Value.List` altogether on the type system level?

Reply:
Yes.

Reply:
No, please keep it. The upcoming histogram needs it.

Reply:
okay, so merging the PR as is. Thx!